### DISCUSSION TITLE
Remove resource existence indication where the data isn't in a scan format - Work page

### DISCUSSION THREAD

**Discussion by wujastyk** [PanditProject/Website#15] [Help] [answered]
A while back, the main listing of MSS in a work entry had a black dot to mark MSS for which there was a link to a scan of the MS.  The black dot now links to a MS entry that has any URL link attached to it.  This is not the desired behaviour, as per our former discussions about this dot feature.

E.g. in the following display, the first dot appears because of a DOI link to a research article.  The second dot appears because there's an online scan of the Chandigarh MS.

![image](https://user-images.githubusercontent.com/762246/118874683-685eec80-b8a8-11eb-9adc-4a46a7d507c5.png)

**Comment by AmirSimantov** [comment]
It is indeed complex, and we may either found an edge case or should revise this feature.

We added this new feature to your request (and then revised it a few weeks ago); it seems to work well. Let's see how we can improve it :)

First, an explanation (we should document it properly later):
On the Work page, inside the box "Related manuscripts," we add that dot if there are any data resources for a Manuscript entity.
We consider a resource to be a "data" one if the value under the "Manuscript -> Resources -> Data text format" inner field is anything but "Metadata only." That is if the value is any of "Scan encoded," "Scan," or "Electronic."

<img width="422" alt="Manuscript -  Resources -  Data text format -  Options" src="https://user-images.githubusercontent.com/10967743/118903682-e8537980-b8dd-11eb-8171-d1b86307e4f5.png">

In the example you supply, the first dot appears **not** because of the link. Rather, it's because the value under the mentioned "Data text format" inner field is "Electronic" (you have added it [6 days ago](https://www.panditproject.org/entity/102310/manuscript#overlay=node/102310/revisions/view/193640/195103)).

I have checked the resource (a PDF file that is displayed online [here](https://indianmedicine.nl/article/view/37385/34966)), and I see that indeed there is some data of the manuscript, such as colophons.

@wujastyk, would you recommend **_not_** displaying the data resource indication in the Work page only when the value in the "Data text format" inner field is "Electronic?" Please consider other scholars who might want to know whether there is **_any_** data, even just as electronic text, not just scans.

**Comment by wujastyk** [comment, marked as answer]
Just display a marker in the MS listing when there is a scan of the MS
available.  Nothing else is required.

It is not necessary to have a special marker for anything else.  Users can
find full data by consulting the MS entry itself.  That's what they expect.

The distinctions you make about metadata only, scan encoded, scan, electronic are meaningful to you, but they will not signify anything to a manuscript specialist.  Nobody is going to understand or make correct use of these distinctions.  When you say 
>We consider a resource to be a "data" one if the value under the "Manuscript -> Resources -> Data text format" inner field is anything but "Metadata only." That is if the value is any of "Scan encoded," "Scan," or "Electronic."

it is almost literally meaningless to me.  I'm sorry to be so blunt, but I think it's best to be honest.  :-)  And I'm generally quite good at this stuff :-)

Best,
Dominik

**Reply by AmirSimantov** [reply]
I can understand now and also talked to @ybronner about it that the most important thing is the scan, not the mere fact that there is data. So, as you suggested, we will supply the indication only when there is a scan (either encoded or not).

**Comment by wujastyk** [comment]
Great, thanks!
